contracts/Tokenomics.sol

```diff
@@ -1173,6 +1206,11 @@ contract Tokenomics is TokenomicsConstants {
         // we still record the amount of OLAS allocated for component / agent owner top-ups from the inflation schedule.
         // This amount will appear in the EpochSettled event, and thus can be tracked historically
         uint256 accountTopUps = incentives[5] + incentives[6];
+        // Record donation power state in a settled epoch
+        if (mapDonationPoints[eCounter].totalDonationPower > accountTopUps) {
+            mapDonationPoints[eCounter].moreDonationPower = true;
+        }
+        mapDonationPoints[eCounter].totalDonationPower = uint96(accountTopUps);
```

Comment:
This is a bug, here it must be the `totalUnitTopUps` value, which is finalized at the checkpoint.